Libs/MRML/DisplayableManager/vtkMRMLSliceIntersectionWidget.h

```diff
@@ -209,6 +212,8 @@ class VTK_MRML_DISPLAYABLEMANAGER_EXPORT vtkMRMLSliceIntersectionWidget : public
 
   bool ProcessZoomSlice(vtkMRMLInteractionEventData* eventData);
 
+  bool ProcessIncDecSlice(vtkMRMLInteractionEventData* eventData);
```

Comment:
`IncDec` wasn't clear to me on first reading.  Maybe better to call this `Scroll` mode (this would need to be changed everywhere).